// @discovery-decrypted/schemas/lib/schemas.js
'use strict';
Object.defineProperty(exports, "__esModule", { value: true });
exports.schemas = void 0;
const contributors_schema_1 = require("./contributors/contributors-schema");
const course_schema_1 = require("./courses/course-schema");
const courses_schema_1 = require("./courses/courses-schema");
const learners_schema_1 = require("./learners/learners-schema");
const project_schema_1 = require("./projects/project-schema");
const projects_schema_1 = require("./projects/projects-schema");
const quest_schema_1 = require("./quests/quest-schema");
const quests_schema_1 = require("./quests/quests-schema");
const repo_schema_1 = require("./repos/repo-schema");
const repos_schema_1 = require("./repos/repos-schema");
const tag_schema_1 = require("./tags/tag-schema");
const tags_schema_1 = require("./tags/tags-schema");
exports.schemas = {
    discovery: {
        projects: projects_schema_1.ProjectsSchema,
        project: project_schema_1.ProjectSchema,
        courses: courses_schema_1.CoursesSchema,
        course: course_schema_1.CourseSchema,
        quests: quests_schema_1.QuestsSchema,
        quest: quest_schema_1.QuestSchema,
        contributors: contributors_schema_1.Contributors,
        learners: learners_schema_1.Learners,
        repos: repos_schema_1.ReposSchema,
        repo: repo_schema_1.RepoSchema,
        tags: tags_schema_1.TagsSchema,
        tag: tag_schema_1.TagSchema,
    }
};
